hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/consumer/sender/http/JettyHttpMessageSenderProvider.java

```diff
@@ -17,19 +17,19 @@
     private final HttpClient httpClient;
     private final EndpointAddressResolver endpointAddressResolver;
     private final int requestTimeout;
-    private final DefaultHttpTraceIdAppender traceIdAppender;
+    private final DefaultHttpMetadataAppender metadataAppender;
```

Comment:
I think it would be nice if we use an interface ```MetadataAppender<Request>``` here instead of concrete (and default) implementation